pubspec.yaml

```diff
@@ -38,6 +38,8 @@ dev_dependencies:
   pedantic: ^1.9.0
   flutter_test:
     sdk: flutter
+  flutter_driver:
+    sdk: flutter
```

Comment:
Why this?

Reply:
Sorry, I must have added this also to the package pubspec.yml by accident. The dependency has been removed. :-)